diff -Nur gnome-panel-3.6.2-old/applets/clock/clock.c gnome-panel-3.6.2/applets/clock/clock.c
--- gnome-panel-3.6.2-old/applets/clock/clock.c	2013-05-23 22:16:26.090083334 +0200
+++ gnome-panel-3.6.2/applets/clock/clock.c	2013-05-23 22:17:00.391106235 +0200
@@ -54,7 +54,6 @@
 #define GNOME_DESKTOP_USE_UNSTABLE_API
 #include <libgnome-desktop/gnome-wall-clock.h>
 
-#include <libgweather/gweather-xml.h>
 #include <libgweather/location-entry.h>
 #include <libgweather/timezone-menu.h>
 #include <libgweather/gweather-enum-types.h>
